### DISCUSSION TITLE
can we use firebird-driver with sqlalchemy-firebird?

### DISCUSSION THREAD

**Discussion by albymuktiono** [pauldex/sqlalchemy-firebird#40] [Q&A] [answered]
because requirements are :
fdb
SQLAlchemy>1.3.16
So , we can not use  firebird-driver 1.5.1 https://pypi.org/project/firebird-driver/ with sqlalchemy-firebird.
is it correct?
sorry for my bad english
thanks

**Comment by pauldex** [comment, marked as answer]
Yes, that is correct, the fdb driver is currently the only supported driver.  I plan to start a firebird-driver branch for sqlalchemy-firebird within the next few weeks.

**Comment by pauldex** [comment]
You can use firebird-driver with version 2 of sqlalchemy-firebird
